apps/els_lsp/src/els_dialyzer_diagnostics.erl

```diff
@@ -66,9 +66,10 @@ source() ->
 %%==============================================================================
 %% Internal Functions
 %%==============================================================================
--spec diagnostic({any(), {any(), integer()}, any()}) ->
+-spec diagnostic({any(), {any(), erl_anno:anno()}, any()}) ->
         els_diagnostics:diagnostic().
-diagnostic({_, {_, Line}, _} = Warning) ->
+diagnostic({_, {_, Anno}, _} = Warning) ->
+  Line     = erl_anno:line(Anno),
```

Comment:
improvement idea: what if column value would be utilized as well if available?
(I dont want to mention the smartness of the compiler diagnostics which looks for a matching POI to even set the end location)

Reply:
Good idea! 
Maybe we should merge this since dialyzer diagnostics are currently broken andimprove this later. o

Reply:
Fixed in #1007